<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanetaryImagerMainWindow</class>
 <widget class="QMainWindow" name="PlanetaryImagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planetary Imager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="image" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <widget class="QMenu" name="menu_device_load">
     <property name="title">
      <string>&amp;Load...</string>
     </property>
    </widget>
    <addaction name="action_devices_rescan"/>
    <addaction name="menu_device_load"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuPanels">
    <property name="title">
     <string>Pa&amp;nels</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionChip_Info"/>
    <addaction name="actionCamera_Settings"/>
    <addaction name="actionRecording"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionHide_all"/>
    <addaction name="actionShow_all"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionEdges_Detection"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Config&amp;uration</string>
    </property>
    <addaction name="actionShow_settings"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuPanels"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="chipInfoWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Camera Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="camera_settings">
   <property name="windowTitle">
    <string>Settin&amp;gs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="settings_container">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>80</width>
          <height>153</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="recording">
   <property name="windowTitle">
    <string>&amp;Recording</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
  </widget>
  <widget class="QToolBar" name="imageManipulation">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEdges_Detection"/>
   <addaction name="actionShow_settings"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_ROI"/>
   <addaction name="actionClear_ROI"/>
   <addaction name="actionTrack"/>
  </widget>
  <widget class="QDockWidget" name="histogram">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Histogra&amp;m</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
  </widget>
  <action name="action_devices_rescan">
   <property name="text">
    <string>&amp;Rescan</string>
   </property>
  </action>
  <action name="action_device_use">
   <property name="text">
    <string>&amp;Load...</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="actionChip_Info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Chip Info</string>
   </property>
  </action>
  <action name="actionCamera_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera &amp;Settings</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Recording</string>
   </property>
  </action>
  <action name="actionStart_Live">
   <property name="text">
    <string>Start Live</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionPickFolder">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/folder.png</normaloff>:/resources/folder.png</iconset>
   </property>
   <property name="text">
    <string>Pick</string>
   </property>
  </action>
  <action name="actionHide_all">
   <property name="text">
    <string>&amp;Hide all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionShow_all">
   <property name="text">
    <string>Sh&amp;ow all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionShow_settings">
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Show settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Planetary Imager</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionEdges_Detection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/action_edges_detection.png</normaloff>:/resources/action_edges_detection.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edges Detection</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>H&amp;istogram</string>
   </property>
  </action>
  <action name="actionSelect_ROI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select ROI</string>
   </property>
  </action>
  <action name="actionClear_ROI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear ROI</string>
   </property>
  </action>
  <action name="actionTrack">
   <property name="text">
    <string>Track</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
